packages/react-server-examples/hello-world/package.json

```diff
@@ -17,8 +17,8 @@
     "babel-runtime": "~6.5",
     "react": "~0.14.2",
     "react-dom": "~0.14.2",
-    "react-server": "~0.2.0",
-    "react-server-cli": "~0.2.0",
+    "react-server": "~0.4.2",
+    "react-server-cli": "~0.4.2",
```

Comment:
Yikes.  We need a way to keep these up to date automatically.  Maybe https://github.com/lerna/lerna/issues/126?